src/components/application_manager/test/state_controller/state_controller_test.cc

```diff
@@ -131,7 +134,11 @@ class StateControllerImplTest : public ::testing::Test {
       , usage_stat("0",
                    utils::SharedPtr<usage_statistics::StatisticsManager>(
                        new usage_statistics_test::MockStatisticsManager))
-      , applications_(application_set_, applications_lock_) {}
+      , applications_(application_set_, applications_lock_)
+      , message_helper_mock_(
+            *application_manager::MockMessageHelper::message_helper_mock()) {
+    Mock::VerifyAndClearExpectations(&message_helper_mock_);
```

Comment:
@LitvinenkoIra Please also add the Mock::VerifyAndClearExpectations(&message_helper_mock_); call to destructor for this class.